AdobeMediaGalleryApi/Api/Data/AssetInterface.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeMediaGalleryApi\Api\Data;
+
+use Magento\Framework\Api\ExtensibleDataInterface;
+
+/**
+ * Asset Interface
+ *
+ * @api
+ */
+interface AssetInterface extends ExtensibleDataInterface
+{
+    const ID = 'id';
```

Comment:
Once hydrator is applied to `getById` command, these constants are going to be used only inside the Asset model, so it's better to move them to the model and make private